test/integration/index.js

```diff
@@ -325,9 +304,8 @@ before(async function() {
 	const testPack = lib.ModPack.fromJSON({});
 	testPack.id = 12;
 	testPack.name = "subspace_storage-pack";
-	testPack.factorioVersion = "1.1.0";
-	testPack.mods.set("clusterio_lib", { name: "clusterio_lib", enabled: true, version: "0.1.2" });
-	testPack.mods.set("subspace_storage", { name: "subspace_storage", enabled: true, version: subspaceStorageVersion });
+	testPack.factorioVersion = "2.0.0";
```

Comment:
I am surprised this does not cause errors when the tests are still using factorio 1.1. I would expect this to fail so may need to be looked into. Either way, no action here for this PR.

Reply:
The testpack is mostly used for unit tests, it doesn't actually load factorio (at least for most of them)